### ISSUE TITLE
Highlighted code fence text is invisible on Paradox Phantom

### ISSUE THREAD

**Issue by leahpenn** [george-paul/typoratheme-paradox#2]
when I write down like this: 
==normal highlight and `normal code`==
that `normal code` is invisible.
I think this is an issue caused by the font color. It should've been *WHITE* or any other *LIGHT COLOR*. But when it appears at the same time as the highlights, it becomes invisible.

**Comment by george-paul** [comment]
@Leepin-Von Yeah that was a bit of a mistake :p I'm busy with academics tho so feel free to open a PR. If not, I'll get to fixing it eventually.

Note: Edited your issue title to be more descriptive (which is good practice fyi!)
